WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -150,7 +150,7 @@ private extension OrdersViewController {
     func refreshResultsPredicate() {
         resultsController.predicate = {
             let excludeSearchCache = NSPredicate(format: "exclusiveForSearch = false")
-            let excludeNonMatchingStatus = statusFilter.map { NSPredicate(format: "status = %@", $0.rawValue) }
+            let excludeNonMatchingStatus = statusKeyFilter.map { NSPredicate(format: "status = %@", $0.rawValue) }
```

Comment:
```suggestion
            let excludeNonMatchingStatus = statusKeyFilter.map { NSPredicate(format: "statusKey = %@", $0.rawValue) }
```

@mindgraffiti we need to change the predicate format string to `statusKey` otherwise filtering causes the app to crash. 💣